src/lamp_py/ingestion/compress_gtfs/schedule_details.py

```diff
@@ -15,22 +12,13 @@
 import polars as pl
 
 from lamp_py.runtime_utils.process_logger import ProcessLogger
+from lamp_py.ingestion.utils import (
+    ordered_schedule_frame,
+    file_as_bytes_buf,
+)
 
-from .gtfs_schema_map import gtfs_schema
-
-
-def file_as_bytes_buf(file: str) -> BytesIO:
-    """
-    Create buffer of local or http(s) file path
```

Comment:
move these functions to ingestion utils file to be used by compression and ingestion application